dsc/decisionMaker.md

```diff
@@ -0,0 +1,36 @@
+
+DSC also has monitoring and reporting built in. If a system is no longer compliant, DSC can raise an alert and act to correct the system. 
+
+## Benefits of Using Desired State Configuration ##
+
+Configurations are designed to be easily read, stored, and updated. Configurations simply declare the state target devices should be in, instead of writing instructions for how to put them in that state. This makes it much less costly for to learn, adopt, implement, and maintain configuration through DSC. 
+
+Creating configurations means that complex deployment steps are captured as a "single source of truth" in a single location. This makes repeated deployments of a specific set of machines much less error-prone. In turn, this makes deployments faster and more reliable. This enables quick turnaround on complex deployments.
+
+Configurations are also shareable via the PowerShell gallery. This means common scenarios and best practices might already exist for the work you need done.
+
+
+## Desired State Configuration and DevOps ##
+
+[DevOps](http://blogs.technet.com/b/ashleymcglone/archive/2015/11/20/devops-for-n00bs-ie-windows-people.aspx) is a combination of people, technologies and culture that allow for rapid deployment and iteration. DSC was designed with DevOps in mind. Having a single configuration define an environment means that developers can encode their requirements into a configuration, check that configuration into source control, and operations teams can easily deploy code without having to go through error-prone manual processes. 
+
+Configurations are also [data-driven](https://msdn.microsoft.com/en-us/powershell/dsc/configdata), which makes it easier for ops teams to identify and change environments without developer intervention. 
+
+## Desired State Configuration On and Off Premise ##
+
+DSC can be used to manage both on-premise and off-premise deployments. For on-premise solutions, Desired State Configuration has a [pull server](https://msdn.microsoft.com/en-us/powershell/dsc/pullserver) that can be used to centralize management of machines and report on their status. For cloud solutions, Desired State Configuration is usable wherever Windows is usable. There are also specific offerings from Azure built on Desired State Configuration, such as [Azure Automation](https://azure.microsoft.com/en-us/documentation/services/automation/), which  centralizes reporting of Desired State Configuration. 
+
+## DSC and Compatibility ##
+
+Although DSC was introduced in Windows Server 2012 R2, it is available for downlevel operating systems via the Windows Management Framework (WMF) package. More information about the WMF can be found on the [PowerShell landing page](https://msdn.microsoft.com/en-us/powershell/). 
```

Comment:
`More information about WMF can be found [here](https://msdn.microsoft.com/en-us/powershell/wmf/releaseNotes).`